consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/network/IbftMessageTransmitter.java

```diff
@@ -47,42 +53,57 @@ public void multicastProposal(
       final ConsensusRoundIdentifier roundIdentifier,
       final Block block,
       final Optional<RoundChangeCertificate> roundChangeCertificate) {
-    final Proposal data =
-        messageFactory.createProposal(roundIdentifier, block, roundChangeCertificate);
+    try {
+      final Proposal data =
+          messageFactory.createProposal(roundIdentifier, block, roundChangeCertificate);
 
-    final ProposalMessageData message = ProposalMessageData.create(data);
+      final ProposalMessageData message = ProposalMessageData.create(data);
 
-    multicaster.send(message);
+      multicaster.send(message);
+    } catch (final SecurityModuleException e) {
+      LOG.warn("Failed to generate signature for Proposal (not sent): {} ", e.getMessage());
+    }
   }
 
   public void multicastPrepare(final ConsensusRoundIdentifier roundIdentifier, final Hash digest) {
-    final Prepare data = messageFactory.createPrepare(roundIdentifier, digest);
+    try {
+      final Prepare data = messageFactory.createPrepare(roundIdentifier, digest);
```

Comment:
Same as above for the all the other messages in the message transmitter

Reply:
agreed - but so long as we don't transmit, then we've done what we need to.